src/sentry/integrations/github/webhook.py

```diff
@@ -673,5 +693,12 @@ def handle(self, request: HttpRequest) -> HttpResponse:
             logger.exception("Invalid JSON.")
             return HttpResponse(status=400)
 
-        handler()(event)
+        event_handler = handler()
+
+        with IntegrationWebhookEvent(
+            interaction_type=event_handler.event_type,
+            domain=IntegrationDomain.SOURCE_CODE_MANAGEMENT,
+            provider_key="github",
+        ).capture():
+            event_handler(event)
```

Comment:
Q: this line calls `_handle` on the handler(e.g InstallationEventWebhook)  ?

Reply:
initializing and passing the event will call `__call__` which will eventually call `_handle`